plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/template.examples.test.ts

```diff
@@ -71,7 +71,7 @@ describe('fetch:template examples', () => {
   beforeEach(() => {
     mockDir.clear();
     action = createFetchTemplateAction({
-      reader: Symbol('UrlReader') as unknown as UrlReader,
+      reader: Symbol('UrlReader') as unknown as UrlReaderService,
```

Comment:
Should we update this symbol as well?

Reply:
Eh, bit of a weird pattern to start tbh. Can probably just leave as is I don't think it has any effect